<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>914</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spine Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainWindow.qrc">
    <normaloff>:/icons/icon16.png</normaloff>:/icons/icon16.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="hboxLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="slicePainter" name="painter" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>258</width>
        <height>343</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>258</width>
        <height>343</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QVTKWidget" name="vis" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>256</width>
        <height>256</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_DICOM_series"/>
    <addaction name="actionOpen_initialization"/>
    <addaction name="actionSave_initialization"/>
    <addaction name="actionLoad_polygonal_mesh"/>
    <addaction name="actionSave_slices"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionClear_polygonal_data"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionInteractive_inflation"/>
    <addaction name="actionSave_binary_masks"/>
    <addaction name="actionShow_mask_overlays"/>
    <addaction name="actionShow_wireframe"/>
    <addaction name="actionSave_LH_images_and_histogram"/>
    <addaction name="actionUse_DICOMs_windowing"/>
    <addaction name="actionSave_debug_images"/>
    <addaction name="actionUse_shape_model"/>
   </widget>
   <widget class="QMenu" name="menuFiltering">
    <property name="title">
     <string>Filtering</string>
    </property>
    <addaction name="actionAnisotropic_diffusion"/>
    <addaction name="actionMedian_denoising"/>
    <addaction name="actionGaussian_smoothing"/>
    <addaction name="actionLogarithmic_rescaling"/>
    <addaction name="actionSchlick_URQ_rescaling"/>
   </widget>
   <widget class="QMenu" name="menuTransfer_functions">
    <property name="title">
     <string>Transfer functions</string>
    </property>
    <addaction name="actionReset_TF"/>
    <addaction name="actionOpacityPlus"/>
    <addaction name="actionOpacityMinus"/>
    <addaction name="actionTF_shift_Up"/>
    <addaction name="actionTF_shift_Down"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionQuick_help"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuFiltering"/>
   <addaction name="menuTransfer_functions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_slices">
   <property name="text">
    <string>Save slices...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Screenshot...</string>
   </property>
   <property name="toolTip">
    <string>Take a screenshot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionInteractive_inflation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interactive inflation</string>
   </property>
   <property name="toolTip">
    <string>Display vertebral bodies as they are being inflated</string>
   </property>
   <property name="statusTip">
    <string>Display vertebral bodies as they are being inflated</string>
   </property>
  </action>
  <action name="actionSave_binary_masks">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save binary masks</string>
   </property>
   <property name="toolTip">
    <string>If checked, saves binary mask .mha file for each vertebra in &quot;D:Temp&quot; folder</string>
   </property>
   <property name="statusTip">
    <string>If checked, saves binary mask .mha file for each vertebra in &quot;D:\Temp\&quot; folder</string>
   </property>
  </action>
  <action name="actionAnisotropic_diffusion">
   <property name="text">
    <string>Anisotropic diffusion</string>
   </property>
   <property name="toolTip">
    <string>Apply anisotropic diffusion denoising method to the image</string>
   </property>
   <property name="statusTip">
    <string>Apply anisotropic diffusion denoising method to the image</string>
   </property>
  </action>
  <action name="actionMedian_denoising">
   <property name="text">
    <string>Median denoising</string>
   </property>
   <property name="toolTip">
    <string>Apply median denoising method to the image</string>
   </property>
   <property name="statusTip">
    <string>Apply median denoising method to the image</string>
   </property>
  </action>
  <action name="actionGaussian_smoothing">
   <property name="text">
    <string>Gaussian smoothing</string>
   </property>
   <property name="toolTip">
    <string>Apply gaussian smoothing to the image</string>
   </property>
   <property name="statusTip">
    <string>Apply gaussian smoothing to the image</string>
   </property>
  </action>
  <action name="actionLogarithmic_rescaling">
   <property name="text">
    <string>Logarithmic rescaling</string>
   </property>
   <property name="toolTip">
    <string>Apply logarithmic rescaling to the image (x=log(1+x) voxelwise)</string>
   </property>
   <property name="statusTip">
    <string>Apply logarithmic rescaling to the image (x=log(1+x) voxelwise)</string>
   </property>
  </action>
  <action name="actionSchlick_URQ_rescaling">
   <property name="text">
    <string>Schlick URQ rescaling</string>
   </property>
   <property name="toolTip">
    <string>Schlick uniform rational quantization rescaling algorithm</string>
   </property>
   <property name="statusTip">
    <string>Schlick uniform rational quantization rescaling algorithm</string>
   </property>
  </action>
  <action name="actionShow_wireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Initial wireframe</string>
   </property>
   <property name="toolTip">
    <string>Segmented objects are initially created in wireframe view</string>
   </property>
   <property name="statusTip">
    <string>Segmented objects are initially created in wireframe view</string>
   </property>
  </action>
  <action name="actionOpen_initialization">
   <property name="text">
    <string>Open initialization...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_LH_images_and_histogram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save LH images</string>
   </property>
   <property name="statusTip">
    <string>Save Low and High images and LH histogram</string>
   </property>
   <property name="whatsThis">
    <string>Save Low and High images and LH histogram</string>
   </property>
  </action>
  <action name="actionLoad_polygonal_mesh">
   <property name="text">
    <string>Load polygonal mesh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionQuick_help">
   <property name="text">
    <string>Quick help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionClear_polygonal_data">
   <property name="text">
    <string>Clear polygonal data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionOpacityPlus">
   <property name="text">
    <string>Opacity +</string>
   </property>
   <property name="toolTip">
    <string>Increase opacity</string>
   </property>
   <property name="statusTip">
    <string>Increase opacity</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionOpacityMinus">
   <property name="text">
    <string>Opacity -</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionTF_shift_Up">
   <property name="text">
    <string>TF shift Up</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionTF_shift_Down">
   <property name="text">
    <string>TF shift Down</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionReset_TF">
   <property name="text">
    <string>Reset TF</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="actionOpen_DICOM_series">
   <property name="text">
    <string>Open DICOM series...</string>
   </property>
   <property name="toolTip">
    <string>Open DICOM series from a directory</string>
   </property>
   <property name="statusTip">
    <string>Open DICOM series from a directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionUse_DICOMs_windowing">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use DICOM's windowing</string>
   </property>
   <property name="toolTip">
    <string>Apply DICOM's VOI (values of interest) windowing rescaling (if window information is present in files)</string>
   </property>
   <property name="statusTip">
    <string>Apply DICOM's VOI (values of interest) windowing rescaling (if window information is present in files)</string>
   </property>
  </action>
  <action name="actionSave_initialization">
   <property name="text">
    <string>Save initialization...</string>
   </property>
  </action>
  <action name="actionShow_mask_overlays">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show mask overlays</string>
   </property>
   <property name="toolTip">
    <string>Show mask overlays on 2D slices</string>
   </property>
   <property name="statusTip">
    <string>Show mask overlays on 2D slices</string>
   </property>
  </action>
  <action name="actionSave_debug_images">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save debug images</string>
   </property>
   <property name="toolTip">
    <string>Save all intermediate images  inthe temp folder as they are being calculated</string>
   </property>
   <property name="statusTip">
    <string>Save all intermediate images  inthe temp folder as they are being calculated</string>
   </property>
  </action>
  <action name="actionUse_shape_model">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use shape model</string>
   </property>
   <property name="toolTip">
    <string>Use shape model in the second pass</string>
   </property>
   <property name="statusTip">
    <string>Use shape model in the second pass</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>slicePainter</class>
   <extends>QWidget</extends>
   <header>slicePainter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
